test/integration-test/tests/load.rs

```diff
@@ -73,60 +110,108 @@ macro_rules! assert_loaded {
 fn unload_xdp() {
     let bytes = include_bytes_aligned!("../../../target/bpfel-unknown-none/release/test");
     let mut bpf = Bpf::load(bytes).unwrap();
-    let prog: &mut Xdp = bpf
-        .program_mut("test_unload_xdp")
```

Comment:
why rename this? truncation? if so, how did it work before?

Reply:
https://discord.com/channels/855676609003651072/855676609003651075/1128725406313353216 :)